event.go

```diff
@@ -95,17 +97,52 @@ func (e *Event) Fingerprint() []byte {
 // propagated in Chunk. Providing list of events as emitted by transactions in given chunk,
 // hashes should match
 func CalculateEventsHash(es []Event) (crypto.Hash, error) {
+	events, err := sdkEventsToFlow(es)
+	if err != nil {
+		return nil, err
+	}
+
+	id, err := flow.EventsMerkleRootHash(events)
+	if err != nil {
+		return nil, err
+	}
+
+	hash, err := hex.DecodeString(id.String())
+	if err != nil {
+		return nil, err
+	}
+
+	return hash, nil
```

Comment:
not sure what this part is doing, if you need a crypto.Hash, this should work `crypto.Hash(id[:])`

Reply:
Mega facepalm! 🤦 🤦‍♂️ 🤦‍♀️ 
Thank you, fixed